executor/executor-coroutine/src/main/kotlin/com/agoda/boots/executor/CoroutineExecutor.kt

```diff
@@ -0,0 +1,32 @@
+package com.agoda.boots.executor
+
+import com.agoda.boots.Executor
+import com.agoda.boots.Executor.Companion.DEFAULT_CAPACITY
+import kotlinx.coroutines.*
+
+/**
+ * Coroutine implementation of [Executor].
+ *
+ * Can switch to dispatcher to invoke listeners
```

Comment:
There is something wrong with this description. I think it uses dispatcher to launch concurrent bootables only, for listeners and non-concurrent bootables dispatcher is not used in your implementation.

Reply:
Fixed